src/Digdir.Domain.Dialogporten.ChangeDataCapture/ChangeDataCapture/Mappers/DynamicReplicationDataMapper.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.ComponentModel;
+using System.Data;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Reflection;
+using Npgsql;
+using Npgsql.Replication.PgOutput.Messages;
+
+namespace Digdir.Domain.Dialogporten.ChangeDataCapture.ChangeDataCapture.ReplicationMapper;
+
+internal sealed class DynamicReplicationDataMapper<T> : IReplicationMapper<T>
```

Comment:
Dette var det første utkastet til en mapper som funker. Fine med den er at den er generisk, det dumme er at den er ganske kompleks. Bestemte meg for å lage OutboxReplicationMapper fordi denne strengt tatt er overkill når vi kun subscriber på OutboxMessage. Burde jeg la den ligge eller kill it with 🔥?

Reply:
KISS og slikt, så det er overkill for øyeblikket, men implementasjonen her er jo ganske snedig da. Jeg stemmer for å beholde i kodebasen, selv om vi ikke bruker per i dag. (Selv om det bryter med YAGNI da.)